src/cuda/cuda-utils.cpp

```
@@ -30,16 +32,16 @@ ContextScope::~ContextScope()
         CUcontext ctx;
         SLANG_CUDA_ASSERT_ON_FAIL(cuCtxPopCurrent(&ctx));
 #if SLANG_RHI_ENABLE_CUDA_CONTEXT_CHECK
```

Cppcheck (2.20.0)

[style] The function 'create' is never used.

(unusedFunction)